Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG ARCH=amd64
-FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian12
+FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian12@sha256:4a2c1a51ae5e10ec4758a0f981be3ce5d6ac55445828463fce8dff3a355e0b75
```

Comment:
Based on https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#docker, it seems you missed the `org.opencontainers.image.source` label? 

See the example in https://github.com/dependabot-fixtures/docker-with-source/blob/main/Dockerfile also.

Reply:
It seems that you are right after re-reading the doc.